markdown/blog/vsasyncapi-preview-extension.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'AsyncAPI VS Code Preview Goes Web: Now Available in Your Browser'
+date: 2025-12-07T06:00:00+01:00
+type: Community
+tags:
+  - vscode-extension
+  - webview
+cover: /img/posts/vs-asyncapi-preview/image.webp
+authors:
+  - name: Ruchi Pakhle
+    photo: /img/avatars/ruchip16.webp
+    link: https://twitter.com/Ruchicodess
+    byline: AsyncAPI Maintainer & TSC Member
+excerpt: 'AsyncAPI VS Code Preview extension is now web-compatible. Preview AsyncAPI files directly in your browser using vscode.dev or github.dev.'
+---
+
```

Comment:
NGL, the article heavily resembles the one written by AI, because almost all the sections overlap with each other and telling basically the same things over and over again with a slightly different wording.

All the info in the sections below can be combined, made more concise and coherent to better express the idea of how the feature of web compatibility benefits the extension, different categories of users, and AsyncAPI in general:

- With this release, the extension is now fully web-compatible. That means ... 
- What Changed Under the Hood?
- Why Does This Matter?
- Spreading the Word
- What's Next?

FYI @thulieblack